Packs/GitLab/Playbooks/Gitlab_-_Guest_user_permission_change.yml

```diff
@@ -601,7 +601,7 @@ tasks:
       Tag:
         simple: Bad Account
       UserVerification:
-        simple: "True"
+        simple: ${inputs.UserVerification}
```

Comment:
Can you confirm if this was intended?
In some we have hardcoded values and in others a playbook input.

Reply:
Yes, it's fine and approved by Yarden.